css/index.css

```
@@ -773,4 +772,215 @@ body {
 
 .food-spin__location-indicator{
   stroke: var(--title-color);
-}
\ No newline at end of file
+}
+
+.cart-summary {
+  max-width: 500px;
+  background-color: var(--color-white);
+  box-shadow: 0 2px 10px rgba(0, 0, 0, 0.1);
+  position: fixed;
+  top: 0%;
+  right: 0;
+  z-index: 1000;
```

Por que 1000 y no un valor menor como por ej `3`?
```suggestion
  z-index: 3;
```